advice-dont-ask-to-ask.md

```diff
@@ -0,0 +1,50 @@
+# Don't ask to ask[^1]
+
+You may have been sent this document because **you asked to ask** a question,
+such as:
+
+> You: Can someone message me about `<topic>`?<br/>
+> You: Can anyone help me with `<topic>`?<br/>
+> You: Does anyone know about `<topic>`?
+
+This isn't particularly productive and doesn't often lead to others being able to
+help you.
+
+> [!TIP]
+> **tl;dr:** don't ask "Does anyone know about `<topic>`?", but rather ask:
+> - "How do I solve `<problem>` given `<context>`?"
+> - "Why is `<problem>` occurring when I do `<action>`?"
+> - "What do I do when I see `<issue>`, after I've done `<context>`?"
+
+There are plenty of reasons why people who otherwise would help you, may not be able
+to or may not commit to replying to you. If you ask to ask, you're omitting information
+about your problem or question, since you haven't formalized it. You are instead asking
+for something more broad and a large commitment from strangers.
+
+Questions like the above cause conflict and problems because they:
+- ask others to take responsibility to take the lead
+- question people's confidence and knowledge
+- wall out people who might be able to answer your question
+- ask for someone's undivided attention in real time
+- prompt people to commit to interviewing you to get to your real questions
+- obscure any replies, which prevents it from being useful to other people
+
+Others might see this as you not having done the work to diagnose your problem - and
+conclude, why should they now put in any work to help you solve it.
+
+**Instead, take the time to form a question please.**
+
+The solution to the problem is not to ask to ask, but just to ask. Someone may see your question,
+and if they have an idea or recognize the problem, chances are they'll reply.
+
+**If you're not sure what to ask, describe your issue.**
+
+> Can someone help me with `<topic>`?<br/>
```

Comment:
This is definitely an example of how _not_ to ask. I think these examples can be removed actually since we already have good examples in the Tl;DR

Reply:
This quote is meant as one full message.
The problem here is it's probably hard to ensure this is all read as one message, but I can remove the newlines maybe.

I think this is still important since it tells people to write down:
- the topic,
- the problem,
- what they tried,
- what they expected to see,
- what they saw instead
So this is meant to be used as a full template